### DISCUSSION TITLE
Can Crawlee use Redis to build a distributed crawler?

### DISCUSSION THREAD

**Discussion by oldsiks** [apify/crawlee-python#537] [Q&A]
I think the Crawlee framework has a great architecture, but the documentation is quite disorganized, and the examples are not sufficient. I have some specific requirements that can only be understood by reading the source code, which is very inefficient. Additionally, there are no flowcharts, making it somewhat complex to understand.

**Comment by vdusek** [comment]
Hi @oldsiks, we acknowledge that the current documentation needs improvement, and we've been actively working on enhancing it. If you'd like to stay updated on our progress, feel free to check out the [issues labeled "documentation"](https://github.com/apify/crawlee-python/issues?q=is%3Aopen+is%3Aissue+label%3Adocumentation) in the repository.
